app/containers/App/sagas.js

```diff
@@ -119,6 +134,155 @@ export function* validateTokenSaga() {
   }
 }
 
+export function* saveEntitySaga({ data }) {
```

Comment:
@tim-field let me know what you think about these global entity create/update sagas and if/how you think they can be improved